client/src/layout/Footer/Footer.js

```diff
@@ -1,7 +1,272 @@
+  },
+  {
+    imagePath: './assets/images/footer/CHERNIVTSI.jpg',
+    altDescription: 'recent_trips_image',
+  },
+  {
+    imagePath: './assets/images/footer/IVANOFRANKIVSK.jpg',
+    altDescription: 'recent_trips_image',
+  },
+  {
+    imagePath: './assets/images/footer/KYIV.jpg',
+    altDescription: 'recent_trips_image',
+  },
+  {
+    imagePath: './assets/images/footer/KYIVCENTER.jpg',
+    altDescription: 'recent_trips_image',
+  },
+  {
+    imagePath: './assets/images/footer/PRYKARPATTIA.png',
+    altDescription: 'recent_trips_image',
+  },
+];
+
+const FooterContainer = styled('div')({
+  color: 'white',
+  backgroundColor: 'black',
+  paddingBottom: '10px',
+});
+
+const FooterWrapper = styled('div')({
+  margin: '0 auto',
+  padding: '30px',
+  maxWidth: '1200px',
+  '@media (min-width: 768px)': {
+    '& > div': {
+      display: 'flex',
+      gap: '30px',
+    },
+  },
+});
+
+const OurAwards = styled('div')({
+  '& > h3': {
+    paddingBottom: '30px',
+    color: 'white',
+  },
+  '& > p': {
+    paddingBottom: '30px',
+    color: 'white',
+  },
+});
+
+const ContactsData = styled('div')({
+  '& > h3': {
+    color: 'white',
+  },
+});
+
+const ContactsInfo = styled('div')({
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'center',
+  gap: '10px',
+  paddingTop: '30px',
+  '& > div': {
+    display: 'flex',
+    '& > a': {
+      textDecoration: 'none',
+      color: 'inherit',
+      marginLeft: '10px',
+    },
+    '& > p': {
+      marginLeft: '10px',
+      marginBottom: '20px',
+      color: 'white',
+    },
+  },
+});
+
+const SocialIcons = styled('div')({
+  display: 'flex',
+  justifyContent: 'left',
+  gap: '15px',
+  paddingTop: '20px',
+  paddingBottom: '40px',
+  '& > div': {
+    '& > svg': {
+      width: '40px',
+      height: '40px',
+      cursor: 'pointer',
+    },
+  },
+});
+
+const RecentTrips = styled('div')({
+  display: 'grid',
+  columnGap: '5px',
+  gap: '10px',
+  gridTemplateColumns: 'repeat(3, 1fr)',
+  marginBottom: '30px',
+  paddingTop: '30px',
+  '& > a >  img': {
+    width: '100%',
+    borderRadius: '5px',
+  },
+  '@media (min-width: 768px)': {
+    gridTemplateColumns: 'repeat(2, 1fr)',
+  },
+  '@media (min-width: 950px)': {
+    gridTemplateColumns: 'repeat(3, 1fr)',
+  },
+});
+
+const FooterNavigation = styled('div')({
+  borderTop: '1px solid rgba($color: #ffffff, $alpha: 0.5)',
+  margin: '0 auto',
+  padding: '30px',
+  maxWidth: '1200px',
+  '@media (min-width: 768px)': {
+    display: 'flex',
+    flexDirection: 'row-reverse',
+    justifyContent: 'space-between',
+    '& > *': { display: 'flex', gap: '30px' },
+  },
+});
+
+const NavigationList = styled('ul')({
+  display: 'flex',
+  justifyContent: 'left',
+  alignItems: 'center',
+  gap: '20px',
+  padding: '0',
+  paddingBottom: '20px',
+  '& > li': {
+    listStyleType: 'none',
+    ' & > a': {
+      textDecoration: 'none',
+      color: 'white',
+      '&:hover': {
+        color: 'blue',
```

Comment:
please write a styled component with access to themes, then use the colors of our theme

```suggestion
      textDecoration: 'none',
      color: theme.palette.primary.contrastText,
      '&:hover': {
        color: theme.palette.secondary.main,
```

Reply:
solved